### PR TITLE
New docs landing page

### LINKED ISSUE #1631: Postman Introduction Chapter in Docs
It will be good to have a Postman introduction chapter in the docs explaining the salient features of Postman. The chapter must talk about the value proposition the tool provides.

### PR DESCRIPTION
**Pull request by SueSmith**
First draft of new docs landing page - to appear at learning center /docs instead of installation guide. Assuming this'll involve some back and forth but opened as a PR as I thought we could have discussions inline this way. I understand the use cases etc are in flux but have just gone by what's on the website and confluence docs for now. Thoughts appreciated! @loopDelicious @kaustavdm @claireriley 

Fixes #1631

### CODE REVIEW

File: src/pages/docs/postman/launching-postman/introduction.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    url: "/docs/postman/launching-postman/sending-the-first-request"
+  - type: link
+    name: "The API workflow"
+    url: "/docs/postman/design-and-develop-apis/the-api-workflow/"
+
+---
+
+Welcome to the Postman docs!
+
+Postman is a collaboration platform for API development. You can use Postman to design, build, and test APIs in conjunction with your teammates, and to support developer adoption for published APIs. The documentation includes info on carrying out these tasks—you will also find learning material on the [blog](https://blog.getpostman.com) and in the [community forum](https://community.getpostman.com/).
```
Comment (tristandenyer) [inline src/pages/docs/postman/launching-postman/introduction.md]: Recommend using complete words, especially for those where English may not be their first language. Change "info" to "information".

File: src/pages/docs/postman/launching-postman/introduction.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+  - type: link
+    name: "Sending the first request"
+    url: "/docs/postman/launching-postman/sending-the-first-request"
+  - type: link
+    name: "The API workflow"
+    url: "/docs/postman/design-and-develop-apis/the-api-workflow/"
+
+---
+
+Welcome to the Postman docs!
```
Comment (tristandenyer) [inline src/pages/docs/postman/launching-postman/introduction.md]: Recommending "Welcome to the documentation for Postman!"

File: src/pages/docs/postman/launching-postman/introduction.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+[Provide feedback on the documentation.](#providing-feedback)
+
+## Development
+
+![Development](https://www.getpostman.com/img/v2/uses-cases/api-first-development-with-postman.svg)
+
+If you're developing an application, carrying out an API-first development project, or integrating an API into an existing project, you can use Postman to facilitate various stages of the development lifecycle.
+
+* Learn the Postman basics, making authenticated requests, grouping them into collections, inspecting responses, and scripting tests:
```
Comment (tristandenyer) [inline src/pages/docs/postman/launching-postman/introduction.md]: Recommend "Learn the Postman fundamentals"

(We use "fundamentals" in other learning materials.

File: src/pages/docs/postman/launching-postman/introduction.md (hunk 1 above)
Comment (SueSmith) [reply src/pages/docs/postman/launching-postman/introduction.md]: @tristandenyer thanks for the recommendations. I agree it's vital to write for people speaking English as a second language, although not sure I see the issue with "info" - it's also a tricky balance to strike in terms of keeping the tone light and friendly, will consider it though!

File: src/pages/docs/postman/launching-postman/introduction.md (hunk 2 above)
Comment (SueSmith) [reply src/pages/docs/postman/launching-postman/introduction.md]: Hm that seems less friendly and a little cumbersome, may reword the intro anyway though - at this stage I'm mostly trying to start a conversation around whether the topics are right for the use cases hence the devrel reviewers.

File: src/pages/docs/postman/launching-postman/introduction.md (hunk 3 above)
Comment (SueSmith) [reply src/pages/docs/postman/launching-postman/introduction.md]: Hm I wouldn't necessarily go by what we use elsewhere unless it's branding language around a feature or whatever, need to word content in a way that makes sense in the context. Again fundamentals is a wee bit less friendly sounding and tone is super important on this landing page to encourage people to get stuck in. Will consider this if (and more likely when) this bit gets reworded based on figuring out the topics..

**Comment by SueSmith** [comment]
Having a rethink in terms of simplifying this so removing reviewers for the minute to avoid you guys wasting time on it, will ping again when it's ready for some input... 😬

**Comment by SueSmith** [comment]
Closing for the minute, hopefully reopen soon.

**Comment by SueSmith** [comment]
OK greatly simplified version of landing page this time lol - thoughts appreciated! @kaustavdm @claireriley

**Pull request merged**